packages/webapp/pages/onboarding.tsx

```diff
@@ -215,6 +215,9 @@ export function OnboardPage(): ReactElement {
               onSuccessfulRegistration={onSuccessfulTransaction}
               isLoginFlow={isLoginFlow}
               className={classNames('w-full', maxAuthWidth)}
+              onAuthStateUpdate={(isLogin) =>
+                setAuth({ isLoginFlow: isLogin, isAuthenticating: true })
```

Comment:
If `isLogin` is false do we then still set `isAuthenticating` to `true`?

Reply:
It can be registration which is still under authenticating, so both are always true for that property.